From ef3f4be174d893eecc95ca070882a5bf2c1520e4 Mon Sep 17 00:00:00 2001
From: buxiaomo <95112082@qq.com>
Date: Mon, 3 Aug 2020 22:54:17 +0800
Subject: [PATCH 5/8] add openwrt to host file

---
 package/base-files/files/etc/hosts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/base-files/files/etc/hosts b/package/base-files/files/etc/hosts
index b7713ebcc6..f1c310d977 100644
--- a/package/base-files/files/etc/hosts
+++ b/package/base-files/files/etc/hosts
@@ -1,4 +1,5 @@
 127.0.0.1 localhost
+172.16.1.1 openwrt
 
 ::1     localhost ip6-localhost ip6-loopback
 ff02::1 ip6-allnodes
-- 
2.27.0

